### ISSUE TITLE
fasteners 0.16.1 installs the "tests" package

### ISSUE THREAD

**Issue by jimorie** [harlowja/fasteners#74]
I have noticed that installing fasteners 0.16.1 also installs the "tests" package for fasteners. This did not happen for 0.16, and seems like an oversight?

This happened to break my tests as the fasteners "tests" package came before my own "tests" package in my PYTHONPATH.

```
$ python3 -m venv venv

$ venv/bin/pip install fasteners==0.16.1
Collecting fasteners==0.16.1
  Using cached https://files.pythonhosted.org/packages/2e/63/b23490eb6cf3928455118ab5f041d6f4b629b82af91871668673d60353cd/fasteners-0.16.1-py2.py3-none-any.whl
Installing collected packages: fasteners
Successfully installed fasteners-0.16.1

$ ls venv/lib/python3.8/site-packages/tests
__init__.py                test_decorators.py         test_lock.py               test_reader_writer_lock.py
__pycache__                test_helpers.py            test_process_lock.py

$ venv/bin/pip install fasteners==0.16
Collecting fasteners==0.16
  Using cached https://files.pythonhosted.org/packages/78/20/c862d765287e9e8b29f826749ebae8775bdca50b2cb2ca079346d5fbfd76/fasteners-0.16-py2.py3-none-any.whl
Collecting six (from fasteners==0.16)
  Using cached https://files.pythonhosted.org/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl
Installing collected packages: six, fasteners
  Found existing installation: fasteners 0.16.1
    Uninstalling fasteners-0.16.1:
      Successfully uninstalled fasteners-0.16.1
Successfully installed fasteners-0.16 six-1.16.0

$ ls venv/lib/python3.8/site-packages/tests
ls: venv/lib/python3.8/site-packages/tests: No such file or directory
```

**Comment by psarka** [comment]
Indeed an oversight, please update to 0.16.2!

**Comment by jimorie** [comment]
> Indeed an oversight, please update to 0.16.2!

I get the same issue with 0.16.2.

**Comment by psarka** [comment]
Fixed, please try 0.16.3.

**Comment by jimorie** [comment]
> Fixed, please try 0.16.3.

Looks better, thanks for the quick fix! :)
